static/js/components/LearnerSearch.js

```diff
@@ -285,6 +287,7 @@ export default class LearnerSearch extends SearchkitComponent {
             field="profile.birth_country"
             operator="OR"
             itemComponent={CountryRefinementOption}
+            include='.+'
```

Comment:
What does this do? Searchkit has documentation on it but their link to Elasticsearch is broken